api/openapi-spec/swagger.json

```diff
@@ -2863,52 +2863,52 @@
       "title": "SubmitOpts are workflow submission options",
       "properties": {
         "dryRun": {
-          "description": "DryRun validates the workflow on the client-side without creating it.\nThis option is not supported in API.",
           "type": "boolean",
-          "format": "boolean"
+          "format": "boolean",
```

Comment:
All of the changes in `*.go` and `*.swagger.json` files are only docstring changes.